onnx/defs/tensor/defs.cc

```diff
@@ -1822,4 +1822,65 @@ ONNX_OPERATOR_SET_SCHEMA(
 
           propagateShapeFromInputToOutput(ctx, 0, 0);
         }));
+
+static const char* Unique_ver11_doc = R"DOC(
+Finds all the unique values (deduped list) present in the given input tensor. This operator returns 3 outputs. 
+The first output tensor 'uniques' contains all of the unique elements of the input, sorted in the same order that they occur in the input.
+The second output tensor 'idx' is the same size as the input and it contains the index of each value of the input in 'uniques'.
+The third output tensor 'counts' contains the count of each element of 'uniques' in the input.
+
+Example:
+  input_x = [2, 1, 1, 3, 4, 3]
+  output_uniques = [2, 1, 3, 4]
+  output_idx = [0, 1, 1, 2, 3, 2]
+  output_counts = [1, 2, 2, 1]
+)DOC";
+
+ONNX_OPERATOR_SET_SCHEMA(
+    Unique,
+    11,
+    OpSchema()
+        .SetDoc(Unique_ver11_doc)
+        .Input(0, "x", "A 1-D input tensor that is to be processed.", "T")
```

Comment:
Please support for general n-D tensors as detailed here:
https://pytorch.org/docs/stable/torch.html?highlight=unique#torch.unique